content/blog/what-is-an-algorithm/index.md

```diff
@@ -0,0 +1,212 @@
+---
+{
+	title: "What's An Algorithm?",
+	description: "A quick introduction into what algorithms are, what they're made of and why they're an important part of understanding how programming languages work",
+	published: '2022-08-26T18:00:00.000Z',
+	authors: ['qarnax'],
+	tags: ['computer science'],
+	attached: [],
+	license: 'cc-by-nc-sa-4',
+}
+---
+
+If you ever used GPS instructions to get somewhere, or followed a recipe book to make apple pie, or even dialed a phone to call a friend, then you already executed what you can call an **Algorithm**.
+
+In simple terms, an algorithm is a set of instructions that lead to a certain result. If you execute it correctly, you get the result you wanted, a delicious apple pie!
+But if you somehow mess up an instruction (like leaving the pie in the oven for 2 hours instead of 1 hour like the recipe book tells you to) then you end up with a bad result, and you would have to start over.
+
+The same thing applies for machines. If you give a computer a clear set of instructions, it’ll execute them in order to reach the desired result. There is, however, a huge difference between telling another person how to solve a certain equation, and telling a computer to do the same thing, because computers have [their own language](https://unicorn-utterances.com/posts/how-computers-speak#hdd), which is very different from human language.
+
+However, if you understand what an algorithm is, and how to break down tasks into a small set of very basic instructions, it will be a lot easier to deal with Programming languages later in your learning journey.
```

Comment:
We probably shouldn't capitalize `Programming` here:

```suggestion
However, if you understand what an algorithm is, and how to break down tasks into a small set of very basic instructions, it will be a lot easier to deal with programming languages later in your learning journey.
```